samples/bluetooth/mtu_update/README.rst

```diff
@@ -0,0 +1,67 @@
+.. _bluetooth_mtu_update_sample:
+
+Bluetooth: MTU Update
+#####################
+
+Q&A:
+****
+
+  Question: What do I have to do to update my ATT MTU?
+
+| Answer: To get an ATT MTU of x octets, do the following:
+| Set :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU` to at least x
+| Set :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE` to at least x + L2CAP header
+  size
+| Ensure the remote side supports the same MTUs.
+
+  Question: I only want to *send* large packets. I don't need to receive large.
+  Do I still need to set :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE`?
+
+Answer: Yes. [#]_ The Bluetooth specification mandates a symmetric MTU for ATT.
+So RX MTU need to be at least the size of TX MTU.
+
+Overview:
+*********
+
+This sample demonstrates the exchange of MTU between two devices to allow a
+large notification to be sent.
+Updating the MTU can be useful to send bigger packets and so have a better
+throughput.
+
+To be able to send a large notification both the server and the client need to
+update their MTU. The MTU is not a negotiated value, the client and the server
+will exchange their MTUs and choose the minimum of the two. Thus the two MTU can
+be set to a different value, but the MTU of the server must be greater or equal
+to the MTU of the client.
+
+According to the Bluetooth specification, [#]_ MTU represent the maximum size of
+SDUs.
+However, in Zephyr, we can assume that it also represents the maximum size of
+the PDUs. Because, in Bluetooth LE, [#]_ unless we are using L2CAP dynamic
+channels, SDUs are not segmented.
+The Kconfig symbol used to configure the size of the TX MTU is
+:kconfig:option:`CONFIG_BT_L2CAP_TX_MTU`. There is no Kconfig symbol to update
+the size of the RX MTU, because Zephyr uses a buffer pool for ACL RX buffers
+coming from the controller.
+The L2CAP RX MTU depend on the maximum size of ACL RX buffers. This value is
+configured by the Kconfig symbol :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE`.
+The resulting L2CAP RX MTU will be the value of this Kconfig symbol minus the
+L2CAP header size.
+
+.. figure:: img/mtu.svg
+     :align: center
+     :alt: Diagram of the MTUs and their corresponding Kconfig symbols
+
+     Diagram of the MTUs and their corresponding Kconfig symbols
+
+.. [#] Bluetooth Core Specification v. 5.3: Vol. 3, Part F, 3.4.2
+.. [#] Bluetooth Core Specification v. 5.3: Vol. 3, Part A, 5.1
+.. [#] Bluetooth Core Specification v. 5.3: Vol. 3, Part A, 7.3
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/mtu_update` in
+the Zephyr tree.
```

Comment:
can you add a section on the hardware setup, e.g. that you need to flash this part to a device ,and that part to another device. 
and a little text about what you're expected to see if it works.